docs/content/en/docs/reference/clusterspec/packages.md

```diff
@@ -0,0 +1,938 @@
+---
+title: "Packages configuration"
+linkTitle: "Packages configuration"
+weight: 10
+description: >
+  Full EKS Anywhere configuration reference for curated packages.
+---
+
+This is a generic template with detailed descriptions below for reference. To generate your own package configuration, follow instructions from [Package Management]({{< relref "../../tasks/packages" >}}) section and modify it using descriptions below.
+
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: PackageBundleController
+metadata:
+  name: eksa-packages-bundle-controller
+  namespace: eksa-packages
+spec:
+  activeBundle: "v1-21-1001"
+  logLevel: 4
+  privateRegistry: public.ecr.aws/eks-anywhere
+  source:
+    registry: public.ecr.aws/eks-anywhere
+    repository: eks-anywhere-packages-bundles
+  upgradeCheckInterval: "24h"
+  upgradeCheckShortInterval: "5h"
+
+---
+apiVersion: packages.eks.amazonaws.com/v1alpha1
+kind: PackageBundle
+metadata:
+  name: package-bundle
+  namespace: eksa-packages
+spec:
+  packages:
+  - name: hello-eks-anywhere
+    source:
+      registry: public.ecr.aws/eks-anywhere
+      repository: hello-eks-anywhere
+      versions:
+      - digest: sha256:da25f5fdff88c259bb2ce7c0f1e9edddaf102dc4fb9cf5159ad6b902b5194e66
+        name: v0.0
+        images:
+        - digest: sha256:da25f5fdff88c259bb2ce7c0f1e9edddaf102dc4fb9cf5159ad6b902b5194e66
+          repository: public.ecr.aws/eks-anywhere
+
+---
+apiVersion: packages.eks.amazonaws.com/v1alpha1
+kind: Package
+metadata:
+  name: my-hello-eks-anywhere
+  namespace: eksa-packages
+spec:
+  packageName: hello-eks-anywhere
+  packageVersion: v0.0.0-f47374093e8a9c48aca8c7a7f06ae185eb7506f3-helm
+  targetNamespace: eksa-packages
+
+```
+
+## PackageBundleController Fields
+### PackageBundleController
+PackageBundleController is the Schema for the packagebundlecontrollers API.
```

Comment:
Should packagebundlecontrollers be singular (packagebundlecontroller)?

Reply:
Yes. Fixed.